lib/logstash/outputs/http.rb

```diff
@@ -113,7 +113,7 @@ def register
   end # def register
 
   def multi_receive(events)
-    send_events(events)
+    send_events(events) if events.size > 0
```

Comment:
it's more of a styling suggestion and up to you, but I would be to use `unless events.empty?` instead

Reply:
of course. 🤦‍♂️